### DISCUSSION TITLE
무빙 블럭 멀티플레이 질문

### DISCUSSION THREAD

**Discussion by yj098** [naverz/zepeto-studio-kor#1099] [5. Q&A]
안녕하세요 멀티플레이와 다양한 강의 영상 올려주셔서 감사 드립니다:)
멀티 플레이 블럭 예제를 보면서 궁금한 점이 생겨서 질문하게 되었습니다

다른 기능 없이 멀티플레이만 구현해보고 싶어서 예제의 하이어라키에 있는 액터들을
모두 옮겼을 때  옮겨온 모든 액더들이 있을 때는 캐릭터가 잘 불러와지지만,

( UI , Multiplay Scripts) 안에 있는 것들을 제외하고 블럭들만 지웠을 때
캐릭터는 불러와 지는 반면 캐릭터가 움직이지 않습니다
왜 그런지 알 수 있을까요??
코딩 초보라 많이 어렵네요 ㅠㅠ

다른 이벤트 추가 없이 유저들이 돌아다닐 수 있는 월드로만 구성해보고 싶은데
이렇게만, 만들시 여러 예제들중 어떤 것을 보면 좋을까요??

(추가적으로 질문 드리고 싶은 것이 있는데 씬에 넣지 않고 에셋 폴더 내부에 C# 스크립트가 있어도 에러가 나나요??)

두서 없지만, 긴글 읽어주셔서 감사드리고 강의 영상 만들어 주셔서 감사합니다
여러번 따라해 보면서 덕분에 처음 보다는 코딩에 대해서 조금 더 쉽게 접근 할 수 있게 되었네요 :) :)

조금이라도 공부 할 때 어떤 것을 알아 가보면 좋을지 힌트라도 얻어 보기 위해 적게 되었는데 길어 졌네요 읽어주셔서 감사합니다 :)

월드 개발자님들 화이팅:)

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 크리에이터님 반갑습니다 :)
먼저 멀티플레이 공간에서 돌아다니는 예제로는 https://github.com/naverz/zepeto-multiplay-example 이 예제를 사용하시면 좋습니다.
캐릭터 동작과 관련된 스크립트들이 멀티플레이의 경우 Multiplay Scripts 폴더 내의 코드에 정의되어 있기 때문에 이 부분을 삭제하면
올바르게 동작하지 않습니다.
각 코드 내에 주석을 읽어보시면서 필요한 부분을 공부하시면 좋을 것 같습니다.

C# 스크립트는 폴더 내에 있어도 되고 오브젝트에 붙어있어도 됩니다만,
패키지 빌드시에 반영되지 않습니다.
그래서 유니티 에디터에 관련된 부분에 한해서 C#스크립트를 사용해도 됩니다.

감사합니다.

**Reply by yj098** [reply]
정말 감사합니다!~ :):) ㅠㅜㅠㅜㅠ 저 예제 찾고 있었어요 ㅎㅎ 감사해요!!
